src/angular-app/languageforge/lexicon/settings/project-settings.component.html

```diff
@@ -27,7 +27,7 @@
                     </div>
                     <div class="form-group row">
                         <label class="col-form-label col-12">Project Owner</label>
-                        <div class="controls col-12 notranslate">
+                        <div id="e2e-test-project-owner" class="controls col-12 notranslate">
```

Comment:
As it was particularly difficult to write a locator for this element, I added an id to the HTML (according to [this article](https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-32-query-html-elements-based-on-attributes-that-are-unlikely-to-change) recommended by @longrunningprocess)